src/lib/child_processes/child_processes.ml

```diff
@@ -452,17 +455,12 @@ let%test_module _ =
           let%bind () = Async.Writer.save lock_path ~contents:"123" in
           let%bind process =
             start_custom ~logger ~name ~git_root_relative_path ~conf_dir
-              ~args:[ "exit" ] ~stdout:`Chunks ~stderr:`Chunks
+              ~args:[ "exit" ] ~stdout:`Lines ~stderr:`Chunks
```

Comment:
This is the fix that make sure we're 100% passing on tests. Previously pipe may receive a batch of multiple lines.